src/features/Nav/Nav.tsx

```diff
@@ -11,7 +11,7 @@ export const Nav = styled.nav<INavProps>`
   top: 0;
   width: 100%;
   border-bottom: ${(props) => props.borderThickness || '1px'} solid
-    ${(props) => props.theme.colors.greyLighter};
+    ${(props) => props.theme.colors.greyLight};
```

Comment:
what is `greyLight`? 🤔 

I added a task to match dashboard colours with Figma names and static repo: HCK-122